docs/src/key-bindings.md

```diff
@@ -174,7 +174,7 @@ See the [tasks documentation](/docs/tasks#custom-keybindings-for-tasks) for more
 #### Editor
 
 | **Command**                      | **Target** | **Default Shortcut**            |
-| -------------------------------- | ---------- | ------------------------------- |
```

Comment:
This change makes columns inconsistent. 2nd and 3rd still have spaces. Generally it seems like a convention to have the spaces there so how would you feel about undoing this change?

Reply:
Sorry my editor autocompleted it fixing